practica4.js

```diff
@@ -68,7 +68,8 @@ exports.warmer_average_temp =(cities) =>{
     let temp_aux=city.main.temp;
     return temp_aux > TemperaturaMedia;
   });
-   return TempMasCalientes;
+  let nombresCiudades = TempMasCalientes.map((city) => city.name);
+  return nombresCiudades.join(", ");
```

Comment:
En el codigo pone que no me reconoce la funcion cuando hago el node main.js , y no se que esta mal exactamente , ya que todos los metodos estan escritos y la funcion aparentemente esta bien.

Reply:
Posiblemente fuera el `this.average_temp(cities)` que el autoCOREctor que por algún motivo que desconozco no lo encuentra.